src/Pages/MathQuotes.js

```diff
@@ -0,0 +1,20 @@
+/* eslint max-len: ["error", { "code": 700 }] */
+
+/* eslint-disable */
```

Comment:
- [ ] I suggest that you hover on the error you are getting there is always a link that can be useful to resolve the issue